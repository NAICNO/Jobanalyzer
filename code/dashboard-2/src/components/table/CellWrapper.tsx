import { Box, StyleProps } from '@chakra-ui/react'
import { ReactNode } from 'react'

interface CellWrapperProps {
  styleProps?: StyleProps
  children: ReactNode
}

export default function CellWrapper({styleProps, children}: CellWrapperProps) {
  return (
    <Box padding={1} {...styleProps}>
      {children}
    </Box>
  )
}
