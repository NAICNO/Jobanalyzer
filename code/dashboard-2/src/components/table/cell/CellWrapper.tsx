import { ReactNode } from 'react'
import { Box, StyleProps } from '@chakra-ui/react'

interface CellWrapperProps {
  styleProps?: StyleProps
  children: ReactNode
}

export const CellWrapper = ({styleProps, children}: CellWrapperProps) => {
  return (
    <Box padding={1} {...styleProps}>
      {children}
    </Box>
  )
}

