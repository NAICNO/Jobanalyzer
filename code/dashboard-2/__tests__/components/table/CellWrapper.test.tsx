import { describe, it, expect } from 'vitest'
import { render } from '@testing-library/react'
import CellWrapper from '../../../src/components/table/CellWrapper'

describe('CellWrapper', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<CellWrapper><></></CellWrapper>);
    expect(asFragment()).toMatchSnapshot();
  });
});
