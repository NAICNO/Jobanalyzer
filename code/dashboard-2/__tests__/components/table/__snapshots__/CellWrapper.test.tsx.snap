// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CellWrapper > renders correctly 1`] = `
<DocumentFragment>
  <div
    class="css-5tigzz"
  />
</DocumentFragment>
`;
